From 6d57b211fd7c36bd8976023a899cf978a769a2b7 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Thu, 06 Jul 2023 13:50:09 +0300
Subject: [PATCH] [SW-151144] gaudi3: save freeze bin and elf file in project git

The Gaudi3 freeze images should be saved in the project .git
to be used in CI/CD + developement, using the images
built from master is a problematic scenario in case
the images are changed due to build tools changes,
like updating the build toolchain for example.

the frozen binary files are built using the snsp 2019.09 Arc toolchain.
the easiest way to create them is to rebuild arcboot from the v1.11.0
branch.

Change-Id: I22dc2411e7b1a8b0459ace18394c5d55e30a6924
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/bin/gaudi3_arc/asic/arcboot_rom.bin b/bin/gaudi3_arc/asic/arcboot_rom.bin
new file mode 100644
index 0000000..b5f8af4
--- /dev/null
+++ b/bin/gaudi3_arc/asic/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/asic/arcboot_rom.elf b/bin/gaudi3_arc/asic/arcboot_rom.elf
new file mode 100644
index 0000000..090526e
--- /dev/null
+++ b/bin/gaudi3_arc/asic/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi3_arc/onix/arcboot_rom.bin b/bin/gaudi3_arc/onix/arcboot_rom.bin
new file mode 100644
index 0000000..85dc690
--- /dev/null
+++ b/bin/gaudi3_arc/onix/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/onix/arcboot_rom.elf b/bin/gaudi3_arc/onix/arcboot_rom.elf
new file mode 100644
index 0000000..4994959
--- /dev/null
+++ b/bin/gaudi3_arc/onix/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi3_arc/pldm/arcboot_rom.bin b/bin/gaudi3_arc/pldm/arcboot_rom.bin
new file mode 100644
index 0000000..b5f8af4
--- /dev/null
+++ b/bin/gaudi3_arc/pldm/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/pldm/arcboot_rom.elf b/bin/gaudi3_arc/pldm/arcboot_rom.elf
new file mode 100644
index 0000000..b453cc5
--- /dev/null
+++ b/bin/gaudi3_arc/pldm/arcboot_rom.elf
Binary files differ
